### ISSUE TITLE
Prevent Banned Users from accessing Dashboard

### ISSUE THREAD

**Issue by crookedneighbor** [Commander-Spellbook/website-v2#205]
Depends on #204 

Update the dashboard layout to prevent user from using the Dashboard if they are banned.

Remove profile links in navbar (and any other link to the dashboard) if the user is banned.

**Comment by pamobazan12** [comment]
Mejora

**Comment by crookedneighbor** [comment]
No longer relevant as the backend system is happening in a separate app now.
